src/libs/translate.js

```diff
@@ -76,7 +76,12 @@ function translateLocal(phrase, variables) {
     return translate(preferredLocale, phrase, variables);
 }
 
+function getPreferredLocale() {
```

Comment:
NAB: For consistency, we're missing a function doc here -- even though its function is obvious. 

Nice optimization!

Reply:
+ If it has a return value it needs a doc but not necessarily a description.